lib/exercism/active_record_citext.rb

```diff
@@ -1,4 +0,0 @@
-# from http://gray.fm/2013/09/17/unknown-oid-with-rails-and-postgresql/
-ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.tap do |klass|
-  klass::OID.register_type('citext', klass::OID::Identity.new)
-end
```

Comment:
Its part of Rails 4.2 release notes also. https://github.com/rails/rails/blob/master/guides/source/4_2_release_notes.md and http://monosnap.com/image/dxPX5fDYpBfdVAyOMdsmHr9ShPDe0y#